docs/quickstart-docker.md

```diff
@@ -0,0 +1,143 @@
+
+        2023-12-14T15:45:16.331171-00:00 0 [Note] [MY-011825] [Xtrabackup] innodb_data_file_path = ibdata1:12M:autoextend 
+
+        2023-12-14T15:45:16.332194-00:00 0 [Note] [MY-011825] [Xtrabackup] innodb_log_group_home_dir = ./ 
+
+        2023-12-14T15:45:16.332208-00:00 0 [Note] [MY-011825] [Xtrabackup] innodb_log_files_in_group = 2 
+
+        2023-12-14T15:45:16.332319-00:00 0 [Note] [MY-011825] [Xtrabackup] innodb_log_file_size = 50331648 
+
+        2023-12-14T15:45:16.338487-00:00 0 [Note] [MY-011825] [Xtrabackup] inititialize_service_handles suceeded 
+
+        2023-12-14T15:45:16.394774-00:00 0 [Note] [MY-011825] [Xtrabackup] Connecting to MySQL server host: localhost, user: root, password: set, port: not set, socket: not set 
+          
+        …
+         
+        2023-12-14T15:45:19.529503-00:00 0 [Note] [MY-012976] [InnoDB] 8.0.34 started; log sequence number 31467030 
+
+        2023-12-14T15:45:19.530105-00:00 0 [Warning] [MY-012091] [InnoDB] Allocated tablespace ID 1 for sys/sys_config, old maximum was 0 
+
+        2023-12-14T15:45:19.530966-00:00 0 [Note] [MY-011825] [Xtrabackup] Completed loading of 3 tablespaces into cache in 0.000821875 seconds 
+
+        2023-12-14T15:45:19.565412-00:00 0 [Note] [MY-011825] [Xtrabackup] Time taken to build dictionary: 0.0343682 seconds 
+
+        2023-12-14T15:45:19.567908-00:00 0 [Note] [MY-011825] [Xtrabackup] starting shutdown with innodb_fast_shutdown = 1 
+
+        2023-12-14T15:45:19.568313-00:00 0 [Note] [MY-012330] [InnoDB] FTS optimize thread exiting. 
+
+        2023-12-14T15:45:20.569333-00:00 0 [Note] [MY-013072] [InnoDB] Starting shutdown... 
+
+        2023-12-14T15:45:20.582272-00:00 0 [Note] [MY-013084] [InnoDB] Log background threads are being closed... 
+
+        2023-12-14T15:45:20.614336-00:00 0 [Note] [MY-012980] [InnoDB] Shutdown completed; log sequence number 31467030 
+
+        2023-12-14T15:45:20.619892-00:00 0 [Note] [MY-011825] [Xtrabackup] completed OK! 
+        ```
+
+        The command runs a Docker container `pxb` from the `percona/percona-xtrabackup:8.0.34` image and mounts two volumes: one from another container named `psmysql`, which contains Percona Server data directory, and another named `backupvol`, which is where the backup files are stored. The command also sets the user to root and prompts the user to enter the password. The command then executes two steps: 
+    
+        * Runs `xtrabackup` with the `--backup` option to copy the data files from `/var/lib/mysql/` to `/backup_8034`
+    
+        * Runs `xtrabackup` with the `--prepare` option to apply the log files and make the backup consistent and ready for restore
+
+    * You can check the Xtrabackup logs with the `docker container logs <container-name>` command.
+
+        For example:
+
+        ```{.bash data-prompt="$"}
+        $ docker container logs pxb
+        ```
+
+        After executing this command, you'll see the full xtrabackup log from `pxb` container.
```

Comment:
This sentence is needed. You've already said you are checking the logs.

Reply:
removed